apps/wpcom-block-editor/src/default/features/rich-text.js

```diff
@@ -1,5 +1,10 @@
 /* global wpcomGutenberg */
-import { RichTextToolbarButton } from '@wordpress/block-editor';
+import {
+	RichTextShortcut,
+	RichTextToolbarButton,
+	// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+	__unstableRichTextInputEvent,
```

Comment:
While this component has the `__unstable` prefix, is in core for a while now and is public and considered stable.